developer-docs-site/docs/guides/use-aptos-explorer.md

```diff
@@ -0,0 +1,265 @@
+---
+title: "Using Aptos Explorer"
+slug: "use-aptos-explorer"
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Using Aptos Explorer
+
+The [Aptos Explorer](https://explorer.aptoslabs.com/) lets you delve into the activity on the Aptos blockchain in great detail, seeing transactions, validators, and account information. With the Aptos Explorer, you may ensure the work you do in the [Aptos Petra Wallet](install-petra-wallet.md) and elsewhere on the blockchain is accurately reflected in Aptos.
+
+The Aptos Explorer provides a one-step search engine across the blockchain to discover details about wallets, transactions, network analytics, user accounts, smart contracts, and more. The Aptos Explorer also offers dedicated pages for key elements of the blockchain and acts as the source of truth for all things Aptos. See the [Aptos Glossary](../reference/glossary.md) for definitions of many of the terms found here.
+
+## Users
+
+The Aptos Explorer gives you a near real-time view into the status of the network and the activity related to the core on-chain entities. It serves these audiences and purposes by letting:
+
+* App developers understand the behavior of the smart contracts and sender-receiver transaction flows.
+* General users view and analyze Aptos blockchain activity on key entities - transactions, blocks, accounts, and resources.
+* Node operators check the health of the network and maximize the value of operating the node.
+* Token holders find the best node operator to delegate the tokens and earn a staking reward.
+
+## Common tasks
+
+Follow the instructions here to conduct typical work in the Aptos Explorer.
+
+### Select network
+
+The Aptos Explorer renders data from all Aptos networks: Mainnet, Testnet, Devnet, and your local host if configured. See [Aptos Blockchain Deployments](../nodes/aptos-deployments.md) for a detailed view of their purposes and differences.
+
+To select a network in the [Aptos Explorer](https://explorer.aptoslabs.com/), load the explorer and use the *Select Network* drop-down menu at the top right to select your desired network.
+
+<center>
+<ThemedImage
+alt="Select Network in Aptos Explorer"
+sources={{
+    light: useBaseUrl('/img/docs/0-explorer-select-network.png'),
+    dark: useBaseUrl('/img/docs/0-explorer-select-network-dark.png'),
+  }}
+/>
+</center>
+
+### Find a transaction
+
+One of the most common tasks is to track a transaction in Aptos Explorer. You may search by the account address, transaction version and hash, or block height and version.
+
+To find a transaction:
+
+1. Enter the value in the *Search transactions* field near the top of any page.
+1. Do not press return.
```

Comment:
Pressing enter now works on the explorer. It will take the user to the page of the first search result entered.

Reply:
That is - you do not need to explicitly click on the search result in the drop down

Reply:
Done!